src/containers/Tenant/Diagnostics/Healthcheck/Details/Details.tsx

```diff
@@ -13,15 +13,15 @@ import i18n from '../i18n';
 const b = cn('healthcheck');
 
 interface DetailsProps {
-    issueTree?: IIssuesTree;
+    issueTrees?: IIssuesTree[];
     loading?: boolean;
     onUpdate: VoidFunction;
 }
 
 export const Details = (props: DetailsProps) => {
-    const {loading, onUpdate, issueTree} = props;
+    const {loading, onUpdate, issueTrees} = props;
 
-    if (!issueTree) {
+    if (!issueTrees) {
```

Comment:
Is seems that `issueTrees` are always and array, but this array could be empty (`[]`). So it better to check `if (!issueTrees || !issuesTrees.length) {`